### PR TITLE
saving in the checkov reports "FAILED" instead of "OPEN"

### PR DESCRIPTION
**Pull request by itai1357**
**By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

## Description
saving in the checkov reports "FAILED" instead of "OPEN"

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my feature, policy, or fix is effective and works
- [x] New and existing tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by ayajbara — APPROVED** [review]

File: checkov/sca_package/output.py [hunk 1]
```diff
@@ -60,7 +60,10 @@ def create_report_license_record(
     package_name = licenses_status["package_name"]
     package_version = licenses_status["package_version"]
     policy = licenses_status["policy"]
-    status = licenses_status["status"]
+    bc_status = licenses_status["status"]
+
+    # renaming the status name from the one in platform's report to be convenient with checkov's-report
+    status = "FAILED" if bc_status == "OPEN" else bc_status
```
Comment (Saarett) [inline checkov/sca_package/output.py:66]: What could `bc_status` be besides OPEN?

**Review by Saarett — COMMENTED** [review]
good catch!

File: checkov/sca_package/output.py (hunk 1 above)
Comment (gruebel) [reply checkov/sca_package/output.py:66]: probably COMPLAINT or is there also a SKIPPED/SUPPRESSED one?

Comment (itai1357) [reply checkov/sca_package/output.py:66]: for now, only OPEN and COMPLIANT

**Review by gruebel — APPROVED** [review]
🍪

File: checkov/sca_package/output.py (hunk 1 above)
Comment (itai1357) [reply checkov/sca_package/output.py:66]: @maxamel am I right?

Comment (maxamel) [reply checkov/sca_package/output.py:66]: Yes, at the moment we only supply OPEN/COMPLIANT

**Pull request merged**